pom.xml

```diff
@@ -420,6 +420,7 @@
           <pluginDescription>Analyze Java, Scala, Closure and JSP code with SpotBugs. ${spotbugs.version}</pluginDescription>
           <pluginClass>org.sonar.plugins.findbugs.FindbugsPlugin</pluginClass>
           <useChildFirstClassLoader>false</useChildFirstClassLoader>
+          <requiredForLanguages>java,scala,jsp,clojure,kotlin</requiredForLanguages>
```

Comment:
Hello @agigleux
Would you mind confirming that this is correct? Reading the annoucment post it was not 100% clear to me if I should have the languages here or the file extensions for the corresponding languages (for instance `kotlin` or `kt`)

Reply:
It should be the language keys, so it should be fine like this.

SonarQube 10.4 was released, so it would be great if you could test it with the feature enabled (it is disabled by default in 10.4) to be 100% sure there are no unexpected side effects.

Reply:
Thanks for confirming! The integration tests are now running on SonarQube 10.4 and I've also added this to the orchestrator installing the servers (for the integration tests):
`.setServerProperty("sonar.plugins.downloadOnlyRequired", "true")`